internal/providers/terraform/ibm/ibm_cos_bucket.go

```diff
@@ -0,0 +1,24 @@
+package ibm
+
+import (
+	"github.com/infracost/infracost/internal/resources/ibm"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func getIbmCosBucketRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "ibm_ibm_cos_bucket",
```

Comment:
this should be ibm_cos_bucket instead. there must be a bug in the generator that's prepending ibm_ again...